platform/uboot/uboot.mk

```diff
@@ -102,6 +122,22 @@ $(PRODUCT_OUT)/bootloader-sd.img: $(UBOOT_BINARY)
 	dd if=/dev/null of=$@ bs=1 count=1 seek=$$(( 2048 * 1024 - 256 * 512 ))
 endif
 
+
+ifeq ($(PRODUCT_BOARD_PLATFORM),rockchip)
+$(PRODUCT_OUT)/bootloader-sd.img: $(UBOOT_BINARY)
+	$(ROCKCHIP_FIRMWARE_DIR)/tools/mkimage -n rk3399 -T rksd -d $(ROCKCHIP_FIRMWARE_DIR)/bin/rk33/rk3399_ddr_933MHz_v1.24.bin $(UBOOT_OUT)/idbloader_externel.img
```

Comment:
AFAIK armbian scripts was used as reference. Please add comment with links to those scripts.

Reply:
Please include line number and commit hash into the URL